### DISCUSSION TITLE
Cannot connect additional sites with Panopticon connector when Admin Tools is enabled.

### DISCUSSION THREAD

**Discussion by beemerrider** [akeeba/panopticon#796] [Q&A (Community Support)] [answered]
Have about a dozen sites added working just fine. Updated Panopticon today and then went to add additional sites and none of the new sites would connect. Ran debug and found Admin Tools blocking the connection on initial connection when adding a site. Found nothing in WAF of admin tools saying it was blocking anything. Disabled Admin Tools and was able to install new sites. Then enabled Admin tools. Set CRON frequency on new sites and they updated the Joomla core with no problems (only item needing update).

Debug info 
HTTP Status	500
HTTP Body	
 mmmmm  m    m mmmmm         mmmmmm                            
 #   "# #    # #   "#        #       m mm   m mm   mmm    m mm 
 #mmm#" #mmmm# #mmm#"        #mmmmm  #"  "  #"  " #" "#   #"  "
 #      #    # #             #       #      #     #   #   #    
 #      #    # #             #mmmmm  #      #     "#m#"   #    

========================================================================================================================

Your site experienced a PHP error while trying to serve the API application. This usually means that a third party
plugin is written incorrectly, breaking Joomla's API application. In other words, the plugin in question is incompatible
with Joomla! 5.1.3 and/or PHP 8.1.29.

The following plugin(s) have been automatically identified as involved in this error condition:

Plugin        : PLG_SYSTEM_ADMINTOOLS
Version       : 7.6.0
Folder        : system
Element       : admintools
Author        : Nicholas K. Dionysopoulos
Author Email  : nicholas@akeeba.com
Author URL    : https://www.akeeba.com

Please remember that you MUST contact the offending plugin's developer, NOT the developers of Akeeba Panopticon. We can
not fix problems in third party code. Remember to include this text in its entirety, *especially* the Detailed
Information below, when asking the plugin's developer for support.

========================================================================================================================

DETAILED INFORMATION
------------------------------------------------------------------------------------------------------------------------

Akeeba\Plugin\System\AdminTools\Utility\BlockedRequestHandler::logRequest(): Argument #2 ($extraLogInformation) must be of type string, null given, called in /home/communitystabili/public_html/plugins/system/admintools/src/Feature/TrackFailedLogins.php on line 57
in [SITE_ROOT]/plugins/system/admintools/src/Utility/BlockedRequestHandler.php(139)

#0 [SITE_ROOT]/plugins/system/admintools/src/Feature/TrackFailedLogins.php(57): Akeeba\Plugin\System\AdminTools\Utility\BlockedRequestHandler->logRequest('loginfailure', NULL, NULL)
#1 [SITE_ROOT]/plugins/system/admintools/src/Extension/AdminTools.php(754): Akeeba\Plugin\System\AdminTools\Feature\TrackFailedLogins->onUserLoginFailure(Array)
#2 [SITE_ROOT]/plugins/system/admintools/src/Extension/AdminTools.php(373): Akeeba\Plugin\System\AdminTools\Extension\AdminTools->runVoidFeature('onUserLoginFail...', Array)
#3 [SITE_ROOT]/libraries/vendor/joomla/event/src/Dispatcher.php(454): Akeeba\Plugin\System\AdminTools\Extension\AdminTools->onUserLoginFailure(Object(Joomla\CMS\Event\User\LoginFailureEvent))
#4 [SITE_ROOT]/libraries/src/Application/CMSApplication.php(928): Joomla\Event\Dispatcher->dispatch('onUserLoginFail...', Object(Joomla\CMS\Event\User\LoginFailureEvent))
#5 [SITE_ROOT]/libraries/src/Application/ApiApplication.php(316): Joomla\CMS\Application\CMSApplication->login(Array, Array)
#6 [SITE_ROOT]/libraries/src/Application/ApiApplication.php(110): Joomla\CMS\Application\ApiApplication->route()
#7 [SITE_ROOT]/libraries/src/Application/CMSApplication.php(306): Joomla\CMS\Application\ApiApplication->doExecute()
#8 [SITE_ROOT]/api/includes/app.php(50): Joomla\CMS\Application\CMSApplication->execute()
#9 [SITE_ROOT]/api/index.php(31): require_once('/home/community...')
#10 {main}

------------------------------------------------------------------------------------------------------------------------
HTTP HEADERS	
Date
Sat, 31 Aug 2024 00:20:48 GMT
Server
Apache
X-Panopticon-Debug
enabled
Connection
close
Transfer-Encoding
chunked
Content-Type
text/plain;charset=UTF-8

**Comment by nikosdion** [comment, marked as answer]
I had already published an update to Admin Tools about 6 hours before you started this discussion. I think that's why you didn't know about it: Joomla cached extension update information for 6 hours.

Update Admin Tools on those sites and you will be able to connect them just fine.
